openpype/hosts/maya/plugins/publish/collect_look.py

```diff
@@ -386,10 +386,13 @@ def collect(self, instance):
 
         self.log.info("Collected resources: {}".format(instance.data["resources"]))
 
-        # Log a warning when no relevant sets were retrieved for the look.
-        if not instance.data["lookData"]["relationships"]:
-            self.log.warning("No sets found for the nodes in the instance: "
-                             "%s" % instance[:])
+        # Log warning when no relevant sets were retrieved for the look.
+        if (
+            not instance.data["lookData"]["relationships"]
+            and "model" not in self.families
```

Comment:
Not sure I understand why are you checking existence of `"model"` in `families` of collector?

Reply:
because the same code can run for looks and models, but only on looks it should issue a warning.

Reply:
Oh `CollectModelRenderSets` inherits from this class. Now it make sense...